src/client/gpm_import_and_canon_name.c

```diff
@@ -2,6 +2,26 @@
 
 #include "gssapi_gpm.h"
 
+static int gpm_name_oid_to_static(gss_OID name_type, gss_OID *name_static)
+{
+#define ret_static(b) \
```

Comment:
This could be coded as looping over an array instead, but I think the compiler is going to do the same thing with it, so it may not be worth the effort to change (i.e., the only thing gained is a couple lines).

Reply:
Eh actually that's what I tried at first, but then you need an initialization function to fill the array, because the content is coming from another library so you can't auto-inizialize at build time, value needs to be pulled in at dynamic linking time...
After looking into it a little bit I decided it was way easier to do it this way.